lua/delaytrain/init.lua

```diff
@@ -14,9 +15,34 @@ local keymaps = {
 
 local is_enabled = false
 
+local function has_val(tab, val)
```

Comment:
Can you remove `has_val` now that it's not being used? That's the last thing, I'm good to merge once you remove this.

Reply:
Oh, sorry, you'll also need to resolve conflicts per my last changes in #4 . You can use my `M.send_keypress` function or change it to use your private `sendkeys`, either is fine. Then I'm good to merge it.

Reply:
Sorry I totally missed your message about the merge, I think it should be good now you can do quick test on your side just in case.

Reply:
Sorry for the delay. This looks good to me, merging now. Thank you!